docs/_data/table/en/operation-command.yml

```diff
@@ -28,6 +28,8 @@ getUsedBytes:
 load:
   Load the data of a file or a directory from under storage into Alluxio.
 loadMetadata:
+  The loadMetadata command is deprecated since Alluxio version 1.1.  Please use "alluxio fs ls
+  <path>" command instead.
```

Comment:
Should we update the documentation of `ls`? Just looking at this it is unclear how I would use `ls` to replace the functionality of `loadMetadata`